modules/game_options.rb

```diff
@@ -0,0 +1,62 @@
+require_relative 'json_games'
+
+module Games
+  def add_games
+    print 'Enter the name of the game: '
+    multiplayer = gets.chomp
+    print 'When was it last played?:  '
+    last_played_at = gets.chomp
+    print 'When was the game published?: '
```

Comment:
[OPTIONAL] For a better user experience, It would be better to mention the date format next to the question to assist the user in entering the date correctly.